updated docs

jamiepratt opened the pull request:
I updated the docs. I thought it made sense to move the instructions about what parameters of config.ts do to comments in config.sample.ts so that the instructions are more easily updated if there are any changes in future.

Martodox commented on docs/_navbar.md:
```diff
@@ -1,2 +1,2 @@
 * [Github](https://github.com/Martodox/animated-graph-generator)
-* [Youtube channel](https://www.youtube.com/playlist?list=PLPvM6BKGmMHePGnWuspLkqHhlb1F_9IJo)
```
Not every video contains the graphs, change to 
https://www.youtube.com/watch?v=kodRKkPA-LU&list=PLPvM6BKGmMHfiwW9m8vXPvLQUmuErDn2f

Martodox commented on docs/youtube-samples.md:
```diff
@@ -2,3 +2,7 @@
+
+## See also
+
+[There are many more such videos on my youtube channel.](https://www.youtube.com/@bartoszjakubowiak)
```
Include yours if you have any uploaded videos!

Martodox commented on src/config.sample.ts:
```diff
@@ -2,35 +2,70 @@ import { subtractHourStrings } from "./helpers/time.js";
       },
     },
   ],
+  // devMode (development mode) true can be helpful for debugging.
```
This will also generate only 1 frame to get a general overview of the graph per section. Saves rendering time and gives an instant feedback

jamiepratt commented:
Rebased onto new master branch.

jamiepratt commented:
addressing feeback from Bartosz

- changed url for his channel to playlist only listing videos with HR data.
- added more info about devMode setting.
- rebased onto new master branch that includes my patch for type issue.
- removed pull request link from Work in Progress sub heading

jamiepratt commented:
It'd be cool to add info about recommended node versions and whatever. Maybe link to an installation tutorial. Or I think you said you could build the whole script so it can be just run from the command line without dependence on npm?
